### PR TITLE
ci: add continuous deployment scripts

### PR DESCRIPTION
**Pull request by TimoGlastra**
Adds continuous deployment scripts for stable releases. We can now run a manual `workflow_dispatch` action on the `Continuous Integration` pipeline and it will create a PR for the next release. 

Once that PR is merged it will automatically tag, create a github release and release to NPM. This is in preparation for the 0.1.0 release I want to do.

### CODE REVIEW

**Comment by TimoGlastra** [comment]
@ryjones I'd like to remove the semantic pull request app from this repo, but it seems I don't have permission to do so. Could you remove the check from this repo? See here for the guide: https://docs.github.com/en/organizations/keeping-your-organization-secure/reviewing-your-organizations-installed-integrations

image

**Comment by ryjones** [comment]
@TimoGlastra I removed it from both this repo the -ext repo.

**Comment by TimoGlastra** [comment]
Thanks! 🙏

File: packages/node/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    ><img
+      alt="License"
+      src="https://img.shields.io/badge/License-Apache%202.0-blue.svg"
+  /></a>
+  <a href="https://www.typescriptlang.org/"
+    ><img
+      alt="typescript"
+      src="https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg"
+  /></a>
+    <a href="https://www.npmjs.com/package/@aries-framework/redux-store"
```
Comment (blu3beri) [inline packages/node/README.md:22]: Is this label displayed I can't seem to find it

File: packages/react-native/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    ><img
+      alt="License"
+      src="https://img.shields.io/badge/License-Apache%202.0-blue.svg"
+  /></a>
+  <a href="https://www.typescriptlang.org/"
+    ><img
+      alt="typescript"
+      src="https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg"
+  /></a>
+    <a href="https://www.npmjs.com/package/@aries-framework/redux-store"
```
Comment (blu3beri) [inline packages/react-native/README.md:22]: Same here.

File: packages/core/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    ><img
+      alt="License"
+      src="https://img.shields.io/badge/License-Apache%202.0-blue.svg"
+  /></a>
+  <a href="https://www.typescriptlang.org/"
+    ><img
+      alt="typescript"
+      src="https://img.shields.io/badge/%3C%2F%3E-TypeScript-%230074c1.svg"
+  /></a>
+    <a href="https://www.npmjs.com/package/@aries-framework/redux-store"
```
Comment (blu3beri) [inline packages/core/README.md:22]: Same here

**Review by blu3beri — APPROVED** [review]

File: packages/core/README.md (hunk 3 above)
Comment (TimoGlastra) [reply packages/core/README.md:22]: Good catch, thanks!

**Pull request merged**